contrib/sssd.spec.in

```diff
@@ -470,6 +472,25 @@ Provides rules for polkit integration with SSSD. This is required
 for smartcard support.
 %endif
 
+%if 0%{?rhel} == 9
```

Comment:
There are build also for rhel 8, so maybe
%if 0%{?rhel} <= 9

Reply:
RHEL8 isn't supported in CI for 'master' branch.